### PR TITLE
feat: Added Home page

### PR DESCRIPTION
**Pull request by mnath30**
Added home page and connected pages using router

Files to Check:
1. Files inside Components folder:
    i. Navbar.jsx
    ii. BannerImage.jsx
    iii. GenderCategoryCard.jsx
    iv. StyleCategoryCard.jsx
    v. Footer.jsx
2. Files inside Pages folder:
    i. Home.jsx
    ii. Error.jsx
3. App.js
4. index.js

### CODE REVIEW

File: src/App.css [hunk 1]
```diff
@@ -2,50 +2,3 @@
...
-  color: #61dafb;
-}
-
-.brand-title {
-  font-size: 3rem;
-  color: #d9d9d9;
-}
-
-.brand-title span {
-  color: #ffde59;
```
Comment (sharathnair9999) [inline src/App.css:36]: you can use variables for colors to maintain uniformity

File: src/App.css [hunk 2]
```diff
@@ -2,50 +2,3 @@
...
-}
-
-.brand-description {
-  font-size: 1.4rem;
-}
-
-.links {
```
Comment (sharathnair9999) [inline src/App.css:43]: Replace one property with a utility class

**Review by sharathnair9999 — APPROVED** [review]
host the images on cloudinary or somewhere and get the data. or page load may take time if you put images into the directory of the project

File: src/App.css (hunk 1 above)
Comment (mnath30) [reply src/App.css:36]: This was the default style present in create react app. I have removed the default styling and it no longer exists

File: src/App.css (hunk 2 above)
Comment (mnath30) [reply src/App.css:43]: This was the default style present in create react app. I have removed the default styling and it no longer exists

File: src/App.js [hunk 3]
```diff
@@ -1,41 +1,20 @@
...
-          <a
-            href="https://github.com/neogcamp/mockBee"
-            target="_blank"
-            rel="noreferrer"
-          >
-            Contribute
-          </a>
-        </div>
-      </header>
+    <div className="App">
```
Comment (spraveenofficial) [inline src/App.js:8]: Remove this unwanted div

File: src/Components/BannerImage/BannerImage.jsx [hunk 4]
```diff
@@ -0,0 +1,21 @@
+import "./banner-image.css" 
+import {img01} from '../../asset';
+
+const BannerImage=()=>{
+    return(
+        <>
```
Comment (spraveenofficial) [inline src/Components/BannerImage/BannerImage.jsx]: I think, here you don't need fragment.

File: src/Components/CategoryCard/GenderCategoryCard.jsx [hunk 5]
```diff
@@ -0,0 +1,19 @@
+import './category-card.css'
+
+const GenderCategoryCard = ({ source, information, caption }) => {
+    return (
+        <>
```
Comment (spraveenofficial) [inline src/Components/CategoryCard/GenderCategoryCard.jsx]: Here too.

File: src/Components/CategoryCard/StyleCategoryCard.jsx [hunk 6]
```diff
@@ -0,0 +1,14 @@
+import "./category-card.css"
+
+const StyleCategoryCard=({source,information,caption})=>{
+    return(
+        <>
```
Comment (spraveenofficial) [inline src/Components/CategoryCard/StyleCategoryCard.jsx]: Here too.

**Review by spraveenofficial — APPROVED** [review]

**Review by Mohitkumar33 — APPROVED** [review]

File: src/Components/CategoryCard/GenderCategoryCard.jsx (hunk 5 above)
Comment (mnath30) [reply src/Components/CategoryCard/GenderCategoryCard.jsx]: Removed all the unnecessary react fragments

**Pull request merged**